packages/app/obojobo-express/config.js

```diff
@@ -85,12 +80,12 @@ const getConfigFileData = (configFile, env) => {
 			envObject = JSON.parse(hydratedJson)
 		}
 
-		return Object.assign({}, defaultObject, envObject) // combine with default if it exists
+		// combine with default if it exists
+		return deepFreeze({ ...defaultObject, ...envObject })
 	} catch (error) {
 		logger.error(`Error loading config file: ${configFile}`)
 		logger.error(error.toString())
-
-		return {}
+		return deepFreeze({})
```

Comment:
These deepFreeze calls may be redundant since it's called again on the whole config object afterwards.  Although - if we expose `getConfigFileData` it would still be useful?